src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -873,11 +873,11 @@ namespace cryptonote
     return true;
   }
   //---------------------------------------------------------------
-  const crypto::view_tag *get_output_view_tag(const cryptonote::tx_out& out)
+  boost::optional<crypto::view_tag> get_output_view_tag(const cryptonote::tx_out& out)
   {
     return out.target.type() == typeid(txout_to_tagged_key)
-      ? &boost::get< txout_to_tagged_key >(out.target).view_tag
-      : NULL;
```

Comment:
Note for future reference: [use](https://www.cprogramming.com/c++11/c++11-nullptr-strongly-typed-enum-class.html) `nullptr` instead of `NULL`.